src/astro/sql/operators/load_file.py

```diff
@@ -76,19 +79,18 @@ def load_data_to_table(self, input_file: File) -> Table:
             ndjson_normalize_sep=self.ndjson_normalize_sep,
             database=database,
         )
-        if_exists = self.if_exists
-        for file in get_files(
+        files = resolve_file_path_pattern(
```

Comment:
@utkarsharma2 , let's use in the operator `database.load_file_to_table`, and let's allow the method   `database.load_file_to_table` to decide if and how to iterate through files if needed. From the Astro Python SDK perspective, `load_file` means wildcard.

As an example, the BigQuery load job supports receiving the wildcard for the files to be matched, so we do not need to iterate through files in this particular path.

Reply:
Moved the logic into database's base class.